packages/subgraph/schema.graphql

```diff
@@ -743,6 +805,7 @@ type SentEvent implements Event @entity {
     operator: Bytes!
     from: Bytes!
     to: Bytes!
+    token: Bytes!
```

Comment:
I didn't see this mapped in: `packages/sdk-core/src/mapGetAllEventsQueryEvents.ts`